share/templates/terminal program/C++/Makefile.func

```diff
@@ -1,6 +1,6 @@
 #
 # Motor IDE functional Makefile template
-# 	for terminal program in C++
+#       for terminal program in C++
```

Comment:
your editor seems to automatically convert tabs into spaces. this causes a lot of unnecessary whitespace noise (check rest of the diff)

Reply:
Vscode does that

Reply:
i don't care who does it, but i care not to merge random whitespace changes :)
it's your task to fix it up (and check whether this behaviour can be disabled in your editor)